src/app/profiles.rs

```diff
@@ -197,10 +197,10 @@ where
                             Ok(Async::Ready(())) => svc.as_service(),
                             Ok(Async::NotReady) => return Ok(Async::NotReady),
                             Err(err) => {
+                                let err = err.into();
```

Comment:
Curious if there is a reason the `err.into()` was moved out of `error!()`?

Reply:
Yes, although it's no longer necessary. In an earlier iteration of this branch where we used the `tokio-trace` log back-compatibility feature to emit `log` records, `into()` would be called twice, once when the expression was used in the log format string and again when it was used as a `tokio-trace` field. We're no longer using the log compat feature so this can be removed.